app/processors/email_processor.rb

```diff
@@ -1,12 +1,25 @@
 class EmailProcessor
   cattr_accessor :default_to_newest_feed, default: false
```

Comment:
`default_to_newest_feed` doesn't seem to influence any behaviour (anymore)?

Reply:
yeah that's true I should fix it